### DISCUSSION TITLE
Penggantian repo di amplify

### DISCUSSION THREAD

**Discussion by estherak23** [awsugid/forum#21] [Q&A]
saya mendapatkan project dari team lain tapi saya tidak mendapatkan repo nya hanya source code nya jadi saya mencoba membuat project baru di amplify ,project terdeploy tapi ui nya saja yg terdeploy data2,foto2 dari backend nya tidak ,dari beberapa stackoverflow katanya karena dns nya belum diubah atau ada settingan yg kurang ,tapi apakah jika dns nya diubah settingan nya akan berantakan atau ada cara lain untuk mengganti repo app yg lama

**Comment by rioastamal** [comment]
Dari aplikasi frontend harusnya terlihat dia melakukan call ke API mana saja dan mengambil assets dari mana saja. Sebaiknya contact team backend yang bersangkutan.
